### PR TITLE
2q dev release

### PR DESCRIPTION
**Pull request by znhafeez**
Epic: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/38041
Issue: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/38051

### CODE REVIEW

File: deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_authentications_crd.yaml [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    kind: Authentication
+    listKind: AuthenticationList
+    plural: authentications
+    singular: authentication
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Authentication is the Schema for the authentications API
```
Comment (yannizhang2019) [inline deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_authentications_crd.yaml:21]: should `Schema` be lower case?

File: deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_oidcclientwatchers_crd.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    kind: OIDCClientWatcher
+    listKind: OIDCClientWatcherList
+    plural: oidcclientwatchers
+    singular: oidcclientwatcher
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: OIDCClientWatcher is the Schema for the oidcclientwatchers API
```
Comment (yannizhang2019) [inline deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_oidcclientwatchers_crd.yaml:21]: `Schema` lower case?

File: deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_authentications_crd.yaml (hunk 1 above)
Comment (znhafeez) [reply deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_authentications_crd.yaml:21]: Yeah `Schema` can be lowercase. We've had it uppercase for the past two csv's.

File: deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_oidcclientwatchers_crd.yaml (hunk 2 above)
Comment (posriniv) [reply deploy/olm-catalog/ibm-iam-operator/3.6.1/operator.ibm.com_oidcclientwatchers_crd.yaml:21]: I don't know why, but, the CRD is named like this `oidcclientwatchers.operator.ibm.com`, though we give the name of the type of oidcclientwatcher. I think `oidcclientwatchers` is all right.

**Pull request merged**